reddit_gold/public/static/css/gold.less

```diff
@@ -760,12 +771,32 @@ section#postcard {
         }
     }
 
+    .selectors .button {
+        border-radius: 0;
+        border-right-width: 0;
+
+        &:first-child {
+            border-top-left-radius: 5px;
+            border-bottom-left-radius: 5px;
+        }
+
+        &:last-child {
+            border-top-right-radius: 5px;
+            border-bottom-right-radius: 5px;
+            border-right-width: 3px;
+        }
+    }
+
     .button-big {
-        font-size: 26px;
-        line-height: 74px;
-        height: 70px;
+        font-size: 18px;
+        line-height: 34px;
+        height: 50px;
         border-color: #ff8415;
```

Comment:
I think they're a little one-off.  Probably what would be better would be to use the colors defined elsewhere for a little more consistency.  I was thinking about this a while ago, the colors defined at the top of markdown.less should really be in a separate file, and these should probably pull from that.  Do you mind if I punt on it for now in the interest of getting it out?